### DISCUSSION TITLE
How to get access token ?

### DISCUSSION THREAD

**Discussion by sriramharibabu** [algo2t/alphatrade#10] [Q&A]
How do we get the access token ? I dont see any code that will automatically assign it so, my assumption is we have to pick it up from the browser and manually paste it everyday .... is my understanding correct?

**Comment by alistair7682** [comment]
@algo2t please guide us and help newbies/beginners on how to apply for api access and how to get access token 

would be grateful to you.

Thanks

**Comment by residue-analytics** [comment]
Hi,
Not sure whether you figured this out or not but the library fetches the token from the successful login response and saves it. You don't have to get the token yourself and create / save it in the access_token.txt file. 
Also, do not attempt login multiple times without analyzing the reason for failure, your account will get blocked which you will have to unblock online (option present on the Alpha Web alpha.sasonline.in).

**Reply by raminfy** [reply]
Yes, I figured out.. Initially it was not working .. after few modifications, I was able to 
Thanks for the reply :-)

**Reply by Tradertarunj** [reply]
Please help me as i am not able to generate access token
